docs_src/design/adr/application/0025-Record-and-Replay.md

```diff
@@ -0,0 +1,128 @@
+# Record and Replay ADR
+
+## Submitters
+- Lenny Goodell (Intel)
+
+## Change Log
+- [Pending](https://github.com/edgexfoundry/edgex-docs/pull/863) (2022-09-20)
+
+## Referenced Use Case(s)
+- [Record and Replay UCR](https://docs.edgexfoundry.org/2.3/design/ucr/Record-and-Replay/)
+
+## Context
+This design involves creating a new Application Service that is responsible for the requirements in the above referenced UCR. This new Application Service will go above and beyond that of a normal Application Service,  thus it is architecturally significant.
```

Comment:
> This new Application Service will go above and beyond that of a normal Application Service,  thus it is architecturally significant.

I don't think creating a complex App Service justifies needing an ADR. I'd just remove this and justify the need for an ADR (as mentioned by Lenny in the TSC call) by saying this is a first-party service and need a formal design review.

Reply:
Updated